client/src/services/content.ts

```diff
@@ -157,15 +157,22 @@ Dit kan worden gebruikt voor:
     },
     footer: {
         column1: `# Contact
-Hebt u een vraag en kunt u het antwoord niet vinden op deze website? Neem dan contact met ons op.`,
+Hebt u een vraag en kunt u het antwoord niet vinden op deze website? Neem dan contact met ons op.
+- [Contactformulier](/)
```

Comment:
Hier moeten we dus nog ergens heen linken ipv `/`.

Reply:
Zie comment direct hierboven. Maar prima om hier - net als op amsterdam.nl - te linken naar https://formulieren.amsterdam.nl/tripleforms/DirectRegelen/formulier/nl-NL/evAmsterdam/Klachtenformulier.aspx